react/core/Navbar/index.tsx

```diff
@@ -0,0 +1,65 @@
+import type { ReactNode } from 'react'
+import type { FlexProps, StyleProp } from '@vtex/admin-ui'
+import { Heading, jsxs, Flex, merge, Button } from '@vtex/admin-ui'
+import { IconArrow } from '@vtex/admin-ui-icons'
+
+export function Navbar(props: NavbarProps) {
+  const navbarProps = useNavbar(props)
+
+  return jsxs({
+    component: Flex,
+    props: navbarProps,
+  })
+}
+
+export function useNavbar(props: NavbarProps): FlexProps {
+  const { styleOverrides, children, title, link, ...rest } = props
+
+  const theme: StyleProp = {
+    height: 76,
+    paddingX: 4,
+  }
+
+  return {
+    styles: merge(theme, styleOverrides),
+    children: [
+      link
+        ? jsxs({
+            component: Button,
+            props: {
+              variant: 'tertiary',
+              onClick: () => link.onClick,
+              icon: jsxs({
+                component: IconArrow,
+                props: {
+                  direction: 'left',
+                  title: link.label,
+                },
+              }),
+            },
+          })
```

Comment:
I like using `jsxs`, but I think in this case it could be a little bit confusing for who is reading the code. Just a suggestion, feel free to accept or not
```suggestion
        ? (
          <Button 
            variant="'tertiary"
            onClick={() => link.onClick}
            icon={<IconArrow direction="left" title={link.label}/>}
        />
      )
```

Reply:
resolved via slack dm. We should add this doc on admin-ui.